qiskit/visualization/pulse_v2/generators.py

```diff
@@ -58,6 +57,31 @@ def my_object_generator(inst_data: InstructionTuple) -> List[drawing_objects.Tex
 # Waveform related information generation
 
 
+def _find_consecutive_index(vector: np.ndarray) -> np.ndarray:
```

Comment:
Would it be better just to return the sparse lists of changes? This couples to the global stylesheet.

Reply:
The problem here is the resolution. This intends to return sparse list by using np.diff, but we also want to ignore tiny changes such as rounding error. But we are not sure if such tiny change is signal or just noise. So the configuration is set in the stylesheet.